sys-utils/unshare.c

```diff
@@ -284,6 +303,8 @@ static void __attribute__((__noreturn__)) usage(void)
 	fputs(_(" -w, --wd=<dir>            change working directory to <dir>\n"), out);
 	fputs(_(" -S, --setuid <uid>        set uid in entered namespace\n"), out);
 	fputs(_(" -G, --setgid <gid>        set gid in entered namespace\n"), out);
+	fputs(_(" --monotonic <offset>      set clock monotonic offset in time namespaces\n"), out);
+	fputs(_(" --boottime <offset>       set clock boottime offset in time namespaces\n"), out);
```

Comment:
Nit: I'd probably clarify it by `--monotonic <offset seconds>`.

Reply:
I included `seconds` in the help text as `<offset seconds>` did not fit in the help text formatting.